content/blog/2017/2017-02-02-scm-api-2-take2.adoc

```diff
@@ -0,0 +1,124 @@
+---
+:layout: post
+:title: SCM API 2.0 Release Take 2
+:tags:
+- development
+- plugins
+:author: stephenc
+---
+
+In January we link:https://jenkins.io/blog/2017/01/17/scm-api-2/[announced the release of SCM API 2.0].
+There were four important issues identified after the original release was published and in order to allow those issues to be resolved, the updates were pulled from the update center.
```

Comment:
Suggestion:
```
After the original release was published we identified four new high-impact issues . 
We decided to remove the new versions of the plugins from the update center until
those issues could be resolved.
```
It's a bit more active voice and "we".   Sounds bit more decisive.